manifests/install.pp

```diff
@@ -6,7 +6,7 @@
   $extra_packages_ensure = $sssd::extra_packages_ensure,
 ) {
 
-  ensure_packages([$sssd_package],{ ensure => $ensure })
+  ensure_packages({$sssd_package => { 'ensure' => $ensure } } )
```

Comment:
This is causing Puppet 3.8.7 with ruby 1.9.3 to fail in a strange way.  need to check on correct syntax as this works on ruby 2.1